sys/include/net/gnrc/netif/internal.h

```diff
@@ -403,7 +404,32 @@ static inline bool gnrc_netif_is_6lbr(const gnrc_netif_t *netif)
 #define gnrc_netif_is_6lbr(netif)               (false)
 #endif
 
+/**
+ * @name    Device type based function
+ *
+ * These functions' behavior is based around the gnrc_netif_t::device_type of
+ * an interface.
+ *
+ * @attention   Special care needs to be taken for those functions when porting
+ *              a new network device type or link-layer protocol: They might
+ *              need adaptions for your port
+ * @{
+ */
+/**
+ * @brief   Get the default link-layer address option for the given
+ *          gnrc_netif_t::device_type of a network interface
+ */
+netopt_t gnrc_netif_get_l2addr_opt(gnrc_netif_t *netif);
```

Comment:
Aren't `@return` and `@param` missing here, or did you let them out intentionally?

Reply:
Good point! I also fixed some weird behavior (and the `const`ness of parameter).